client/src/components/ChallengeCard.tsx

```diff
@@ -11,24 +10,22 @@ function ChallengeCard({
   id,
   difficulty,
 }: ChallengeDetailsShort) {
-
-  const getDifficultyStars = useCallback((difficulty: number) => {
-    return Array.from({ length: difficulty + 1 }, (_, i) => <StarFill key={i} />);
-  }, [difficulty]);
+  const getDifficultyStars = useCallback(
+    (difficulty: number) => {
+      return Array.from({ length: difficulty + 1 }, (_, i) => (
+        <StarFill key={i} />
+      ));
+    },
+    [difficulty],
+  );
 
   return (
     <Card>
-      <Card.Header>
-         {getDifficultyStars(difficulty)}
-      </Card.Header>
+      <Card.Header>{getDifficultyStars(difficulty)}</Card.Header>
       <Card.Body>
         <Card.Title>{title}</Card.Title>
         <Card.Text>{generalDescription}</Card.Text>
-        <Link style={{textDecoration: "none"}} to={"/challenge/" + id}>
-          <Button>
-            Solve
-          </Button>
-        </Link>
+        <Button href={"/challenge/" + id}>Solve</Button>
```

Comment:
The button we have has an href prop that you can use in this situation @Vladmidir

Reply:
When I inspect the webpage in the browser, it does not have the `<a>` anchor tag. I really don't like that. I like to have an anchor tag for navigation.

Reply:
I read up on this. The documentation says that `Providing a href will render an <a> element, styled as a button.`

The only thing is that it still displays as `<button>` in my browser...